src/components/LoanCards/HoverLoanCard/DetailedLoanCard.vue

```diff
@@ -110,8 +110,12 @@
 			<partner-info-panel v-if="hasPartner" :loan-id="loanId" />
 			<trustee-info-panel v-if="hasTrustee" :loan-id="loanId" />
 			<div>
-				<router-link to="">
-					Read full details
+				<hr>
+				<router-link
```

Comment:
Please add tracking to this link. "Lending", "Read full borrower details", loan.id or something like that...

Reply:
heres a better format:
`v-kv-track-event="['Lending', 'click-Read full borrower details', 'Profile Link', loanId, 'true']"`

Reply:
Fixed, true has been removed.